explorer/app/apis/azul/anvil-cmg/common/transformers.ts

```diff
@@ -149,6 +169,17 @@ export function getActivityDataModalities(
   return processAggregatedOrArrayValue(response.activities, "data_modality");
 }
 
+/**
+ * Maps data modalities from aggregated activities values returned from endpoints other than index/activities.
+ * @param response - Response model return from Azul that includes aggregated activities.
+ * @returns Data modalities.
+ */
```

Comment:
I was mostly able to use the same functions that are used for the dataset list, but data modality uses `getActivityDataModalities`, which has an incompatible type, so I created this new `getAggregatedActivityDataModalities` function, which is identical except in input type (and documentation, which I modeled after the other getAggregated functions); should the two data modality functions be combined somehow instead?